libs/clients/national-registry/v1/src/lib/nationalRegistryApi.ts

```diff
@@ -87,6 +89,25 @@ export class NationalRegistryApi {
       : [response.table.diffgram.DocumentElement.ISLFjolskyldan]
   }
 
+  public async getMyChildren(nationalId: string): Promise<ISLBorninMin[]> {
+    const borninMinResponse: GetViewISLBorninMinDto = await this.signal(
+      'GetViewISLBorninMin',
+      {
+        Kennitala: nationalId,
+      },
+    )
+
+    if (!borninMinResponse) {
+      throw new NotFoundException(
+        `children for nationalId ${nationalId} not found`,
+      )
+    }
+
+    const documentData =
+      borninMinResponse.table.diffgram.DocumentElement.ISLBorninMin
```

Comment:
I'd be more defensive here since it's a 3rd party service that might just randomly one day change this/break this and then we might get a null reference. i.e. add as potentially null/undefined in type and check each is available